File: support_tools/swigwin-3.0.12/Examples/test-suite/php/wrapmacro_runme.php
<?php

require "tests.php";
require "wrapmacro.php";

check::functions(array('guint16_swap_le_be_constant', 'maximum'));

check::equal(maximum(2.3, 2.4), 2.4, "maximum() doesn't work");
check::equal(guint16_swap_le_be_constant(0x1234), 0x3412, "GUINT16_SWAP_LE_BE_CONSTANT() doesn't work");

check::done();
?>

File: support_tools/swigwin-3.0.12/Examples/test-suite/go/wrapmacro_runme.go
package main

import "./wrapmacro"

func main() {
	a := 2
	b := -1
	wrapmacro.Maximum(int64(a), int64(b))
	wrapmacro.Maximum(float64(a/7.0), float64(-b*256))
	wrapmacro.GUINT16_SWAP_LE_BE_CONSTANT(1)
}
